### PR TITLE
Rename bet column to description

### PR DESCRIPTION
**Pull request by ghost**
Pivotal: https://www.pivotaltracker.com/story/show/142709805
Renamed all the instances of "bet" that referred to bet text to "bet_description", because the bad naming caused confusion.

### CODE REVIEW

**Review by jacquesporveau — COMMENTED** [review]
LGTM. Only thing that I am going to comment on is the name of your migration which I won't request changes for because I think it's a preference thing. I personally like something along the lines of...

`rename_column_x_in_table_y_to_z.rb:`

File: spec/models/bet_spec.rb [hunk 1]
```diff
@@ -12,28 +12,31 @@
...
   let(:bet1) do
-    Bet.create(bet: 'Superbowl: Atlanta vs New England, place your bets!',
+    Bet.create(bet_description: 'Superbowl: Atlanta vs New England,
+               place your bets!',
                expires_at: DateTime.tomorrow, creator_id: bob.id)
   end
-  let(:option1) { BetOption.create(option_text: 'Atlanta wins', bet: bet1) }
+  let(:option1) do
+    BetOption.create(option_text: 'Atlanta wins', bet: bet1)
+  end
```
Comment (natyork) [inline spec/models/bet_spec.rb]: I try to keep my 'old code cleanup' tasks out of my feature PRs, and submit them as separate PRs :)

File: spec/models/bet_spec.rb [hunk 2]
```diff
@@ -12,28 +12,31 @@
...
   let(:option2) do
     BetOption.create(option_text: 'New England wins', bet: bet1)
   end
   let(:bob_bets_100_on_option1) do
-    UserBet.create(user:       bob,
-                   bet:        bet1,
+    UserBet.create(user: bob,
+                   bet: bet1,
```
Comment (natyork) [inline spec/models/bet_spec.rb]: Code cleanup that could be moved to a separate PR or commit?

File: spec/models/bet_spec.rb [hunk 3]
```diff
@@ -12,28 +12,31 @@
...
                    bet_option: option1,
                    amount_bet: 100)
   end
   let(:peter_bets_50_on_option1) do
-    UserBet.create(user:       peter,
-                   bet:        bet1,
+    UserBet.create(user: peter,
+                   bet: bet1,
```
Comment (natyork) [inline spec/models/bet_spec.rb]: Code cleanup that could be moved to a separate PR or commit?

File: spec/models/bet_spec.rb [hunk 4]
```diff
@@ -12,28 +12,31 @@
...
                    bet_option: option1,
                    amount_bet: 50)
   end
   let(:jack_bets_300_on_option2) do
-    UserBet.create(user:       jack,
-                   bet:        bet1,
+    UserBet.create(user: jack,
+                   bet: bet1,
                    bet_option: option2,
```
Comment (natyork) [inline spec/models/bet_spec.rb]: Code cleanup that could be moved to a separate PR or commit?

File: spec/models/bet_spec.rb (hunk 2 above)
Comment (ghost) [reply spec/models/bet_spec.rb]: Huh, weird, all these spaces shouldn't have existed in the first place, so I guess all this will get resolved with I merge it with all the previous stuff. Thanks for catching it though!

File: spec/models/bet_spec.rb (hunk 1 above)
Comment (ghost) [reply spec/models/bet_spec.rb]: Thanks, didn't really have to fix this one any way, it fit in the 80 character range.

File: db/migrate/20170403171417_rename_bet_to_bet_description_for_bet_model.rb [hunk 5]
```diff
@@ -0,0 +1,5 @@
+class RenameBetToBetDescriptionForBetModel < ActiveRecord::Migration[5.0]
+  def change
+    rename_column :bets, :bet, :bet_description
```
Comment (qr8r) [inline db/migrate/20170403171417_rename_bet_to_bet_description_for_bet_model.rb]: Since we are renaming things........ `bet.bet_description` could probably just be `bet.description`

**Review by qr8r — CHANGES_REQUESTED** [review]

**Comment by jacquesporveau** [comment]
Merge conflicts and I think you have a bunch of commits in here that are unintentional.

**Comment by ghost** [comment]
@jacquesporveau Thanks for pointing it out. That's what happens when you have 5 thousand dependent branches and you update one of them.

**Review by qr8r — APPROVED** [review]

**Review by qr8r — CHANGES_REQUESTED** [review]
I believe this breaks the seeds

**Review by qr8r — APPROVED** [review]
